packages/utils/src/getActiveLoginIDFromLocalStorage.ts

```diff
@@ -1,11 +1,9 @@
 /**
  * Gets the active `loginid` for the current user from the `localStorage`.
  */
-const getActiveLoginIDFromLocalStorage = (loginIDKey?: string) => {
-    const active_loginid = localStorage.getItem('active_loginid');
-    const active_custom_loginid = loginIDKey ? localStorage.getItem(loginIDKey) : undefined;
-
-    return active_custom_loginid ?? active_loginid ?? undefined;
+const getActiveLoginIDFromLocalStorage = (loginIDKey = 'active_loginid') => {
+    const active_custom_loginid = localStorage.getItem(loginIDKey);
+    return active_custom_loginid ?? undefined;
```

Comment:
it would be better to choose one case for variables inside this function. it doesn't make sense to have both sake_case and camelCase variables :)

Reply:
Updated.